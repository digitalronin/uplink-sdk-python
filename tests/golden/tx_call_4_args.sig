r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 58298034293248379837326291052458896311936233417628326794811002244493633578221
