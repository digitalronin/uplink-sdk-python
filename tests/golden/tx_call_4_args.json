{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }, 
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }, 
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }, 
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE4xMDYyMzc0MjIyOTI0NDEzOTA5MTUxMzk2NjQxMjA5ODg5MjM1NDQzNTAzMDYzMTI5NDA5NDEwOTU0MzMyODQ3MjE0MTc4NDEzNjcyNzA="
}